src/artemis/fast_grid_scan_plan.py

```diff
@@ -45,7 +52,7 @@ def run_gridscan(
 
     for id in datacollection_ids:
         run_start(id)
-
+        
```

Comment:
**nit**: Can you remove the superfluous whitespace? I believe vscode should do this for you when you save the file though

Reply:
yes